packages/mineral-ui-tokens/README.md

```diff
@@ -13,39 +12,39 @@ or
 yarn add mineral-ui-tokens
 ```
 
-
 ## Usage
+
 _This package uses the same [import syntax as the mineral-ui library][import-syntax]._
 
 Token names use a `[target]_[property]_[variation]_[state]` naming scheme.
 
-- `target` - the type of element targeted by the token, e.g. "input" or "panel"
-- `property` (required) - the CSS property of the token, e.g. "backgroundColor"
-- `variation` - any differentiating aspect of the token that isn't state, e.g. "brand", "primary", "success"
-- `state` - state-dependent aspects, e.g. "focus", "selected"
-
+* `target` - the type of element targeted by the token, e.g. "input" or "panel"
+* `property` (required) - the CSS property of the token, e.g. "backgroundColor"
+* `variation` - any differentiating aspect of the token that isn't state, e.g. "brand", "primary", "success"
+* `state` - state-dependent aspects, e.g. "focus", "selected"
 
 ### JavaScript
 
 JavaScript token names are formatted in "pseudo_camelCase", e.g. `boxShadow_1`
 
-
 #### Import tokens from the default export:
+
 ```js
 import tokens from 'mineral-ui-tokens';
 ```
 
-#### Import the palette, specific [color ramps][colorPage], or specific tokens from named exports:
+#### Import the palette, specific [color ramps][colorpage], or specific tokens from named exports:
+
 ```js
 import { palette, magenta, boxShadow_1 } from 'mineral-ui-tokens';
 ```
 
 #### Import tokens & palette, as Scss variables:
```

Comment:
Aside from the filename in the import example, this is the only place we use Scss. Probably better to just stick with Sass.

Reply:
Cool; I pulled that from the Readme we created in the first Tokens PR; think I should update it there as well?

Reply:
This is the README. :) But yes, please update both the README and Tokens/index.